src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Color {
```

Comment:
Use the 'name()' method for getting the String representation of enum constants rather than 'toString()' or 'String.valueOf()' to ensure you get the correct name of the enum constant as declared.